### ISSUE TITLE
cred distribution for referrals.

### ISSUE THREAD

**Issue by cheo0202** [ShenaniganDApp/SHEiP#13]
<REMOVE>---
SHEiP: <to be assigned>
title: <cred distribution for referrals.>
author: < jose coronado (@cheo0202)>
discussions-to: <N/A>
status: <Idea >
category: <Standards >
type: <Community >
created: <2021-11-12>
edited: <N/A>
requires: <N/A>
replaces: <N/A>

## Simple Summary
create a referral system with cred awards

## Abstract
The objective is to motivate the community to give it an exponential growth in discord, thus it would help to incorporate the prtcle in the tipcc bot, it would help to share tips between dynamics of the community. You can also help recruit talents to give shenanigan more growth.

## Motivation
My motivation is in the number of projects that use this method to give growth to their communities and motivate them to bring more users and have fun with the tip system.

## Specification
contribute cred to users who invite other users, it would be like 1 cred for every 10 users who invite.

## Rationale
I am based on the cost of including the prtcle in the tipcc bot. since the cost is 1000 $ which that money can go to other priorities of the project. For that reason it occurs to me that the community is in charge of getting referrals to be able to reach 10,000 users and to be able to include the prtcle to the bot for free.

## Implementation
Rewards can start calculating from the day this proposal passes

## Copyright
Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).

**Comment by llealg2** [comment]
tremendous idea to create a referral program, support

**Comment by clna2402** [comment]
I like

**Comment by diegoleal8802** [comment]
the one who brings more wins more excellent

**Comment by youngkidwarrior** [comment]
I think a referral program for BrightID verified users would be great!

That way we would know they are a unique individual.

This proposal lacks the details I would require to make a move, but I'm open to look into this.

**Comment by cheo0202** [comment]
@youngkidwarrior yes it should be with brightid verified users. to take care of multi-accounts

**Comment by yander12** [comment]
it would be good

**Comment by youngkidwarrior** [comment]
@cheo0202 I've moved work on this SHEiP to a [hackmd](https://hackmd.io/fUcG0668SnOJ7mXpFltbDg)

If you'd like to continue this SHEiP, we can work on hackmd, then update this issue as we finalize the idea.
